sdk/Cargo.toml

```diff
@@ -31,9 +31,17 @@ file_io = []
 serialize_thumbnails = []
 no_interleaved_io = ["file_io"]
 fetch_remote_manifests = ["dep:wasi"]
-json_schema = ["dep:schemars", "c2pa-crypto/json_schema"]
-rust_native_crypto = ["c2pa-crypto/rust_native_crypto"]
+json_schema = ["dep:schemars"]
 pdf = ["dep:lopdf"]
+rust_native_crypto = [
+    "dep:const-oid",
+    "dep:der",
+    # "dep:ed25519-dalek",
```

Comment:
Can this line can be deleted?

Reply:
Good catch, thank you. (Usually it's me asking someone else to do exactly that. 😂)